provelo_custom_timesheet_ui/views/hr_timesheet_sheet_views.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="hr_timesheet_sheet_form" model="ir.ui.view">
+        <field name="name">provelo_custom_timesheet_ui.sheet.form</field>
+        <field name="model">hr_timesheet.sheet</field>
+        <field name="inherit_id" ref="hr_timesheet_overtime.hr_timesheet_sheet_form" />
+        <field name="arch" type="xml">
+            <field name="daily_overtime" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </field>
+        </field>
+    </record>
+
+    <record id="hr_timesheet_sheet_tree" model="ir.ui.view">
+        <field name="name">provelo_custom_timesheet_ui.timesheet.sheet.tree</field>
+        <field name="model">hr_timesheet.sheet</field>
+        <field name="inherit_id" ref="hr_timesheet_sheet.hr_timesheet_sheet_tree" />
+        <field name="arch" type="xml">
+            <field name="reviewer_id" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </field>
+            <xpath expr="//tree" position="inside">
+                <field name="timesheet_overtime" invisible="1" />
+            </xpath>
```

Comment:
why does this view not inherit `hr_timesheet_overtime.hr_timesheet_sheet_tree`? this would avoid having to add the `timesheet_overtime` as an invisible field, since it already exists in the view.